scripts/matchPsyToUniversities.ts

```diff
@@ -0,0 +1,88 @@
+import dbUniversities from '../db/universities';
+import dbPsychologists from '../db/psychologists';
+import department from '../utils/department';
+import departementToUniversityName from '../utils/departementToUniversityName';
+
+/**
+ * Update psychologists assigned university according to their department
+ * can be run and rerun again
+ */
+const matchPsyToUni = async (dryRun): Promise<void> => {
+  console.log('Match psychologists to universities...');
+  if (dryRun) {
+    console.log('WARNING: dry-run mode on! (no changes will be applied)');
+  }
+
+  try {
+    const statsAssignmentDone = [];
+    const statsNoDepartmentFound = [];
+    const statsNoUniFound = [];
+    const statsNoChange = [];
+    const universities = await dbUniversities.getAll();
+    const psychologists = await dbPsychologists.getAllAccepted([
```

Comment:
Maybe I shouldn't only do it on accepted psychologists :thinking: I just saw on another trello card that we may want to see assigned universities for refused psychologists too.

Reply:
yes, it cannot hurt to do it for all psy :)